plugins/ua_log_stdout.c

```diff
@@ -50,16 +50,24 @@ __attribute__((__format__(__printf__, 3 , 0)))
 void
 UA_Log_Stdout(UA_LogLevel level, UA_LogCategory category,
               const char *msg, va_list args) {
+#ifdef UNDER_CE
+    UA_DateTimeStruct dts = UA_DateTime_toStruct(UA_DateTime_now());
+#else
     UA_Int64 tOffset = UA_DateTime_localTimeUtcOffset();
     UA_DateTimeStruct dts = UA_DateTime_toStruct(UA_DateTime_now() + tOffset);
+#endif
 
 #ifdef UA_ENABLE_MULTITHREADING
     pthread_mutex_lock(&printf_mutex);
 #endif
 
     printf("[%04u-%02u-%02u %02u:%02u:%02u.%03u (UTC%+05d)] %s/%s" ANSI_COLOR_RESET "\t",
            dts.year, dts.month, dts.day, dts.hour, dts.min, dts.sec, dts.milliSec,
+#ifdef UNDER_CE
+           logCategoryNames[category]);
```

Comment:
This will not be correct since you would also need to change the format string. Is there anything against leaving the original code here and just ignore the fact that in WCE we do not have offset (i.e. it will be 0)